Omulosi/iReporter
# #162686097 Add token revocation endpoint

#### What does this PR do?
Adds an endpoint to revoke a token and hence log out a user from the API
#### Description of Task to be completed?
* create a blacklist table in the database
* Add an endpoint for revoking access tokens
* Add an endpoint for revoking refresh tokens
* update the config file with appropriate settings
#### How should this be manually tested?
clone this repo and checkout the `ft-token-revocation-162686097` branch. Run the application as documented in the README and try logging out a valid access/refresh token. The token should be revoked and should no longer access any endpoint.
#### What are the relevant pivotal tracker stories?
#162686097
